bowtie/_core.py

```diff
@@ -538,6 +543,47 @@ def supports(self, *dialects: Dialect) -> bool:
         """
         return self.info.dialects.issuperset(dialects)
 
+    async def get_versions(self):
+        from github3 import GitHub  # type: ignore[reportMissingTypeStubs]
+        from github3.exceptions import (  # type: ignore[reportMissingTypeStubs]
+            GitHubError,
+        )
+        from github3.models import (  # type: ignore[reportMissingTypeStubs]
+            GitHubCore,
+        )
+        from sortedcontainers import (  # type: ignore[reportMissingTypeStubs]
+            SortedSet,
+        )
+
+        url = CONTAINER_PACKAGES_API / self.id / "versions"
+        versions = SortedSet(key=version_key(reverse=True))
+
+        try:
```

Comment:
A try block should generally have 1 line in it. so you catch only the exception on the line you intend to. Very rarely it's ok for it to have more, but veeeery rarely.

This seems definitely possible to reduce.